src/libcsg/nblistgrid_3body.cc

```diff
@@ -78,7 +78,7 @@ void NBListGrid_3Body::Generate(BeadList &list1, BeadList &list2,
   }
 
   // check if both bead lists "have" the same topology
-  assert(list1.getTopology() == list2.getTopology());
+  assert(&(list1.getTopology()) == &(list2.getTopology()));
```

Comment:
Why is this an `assert`?  Shouldn't if be an runtime error?

Reply:
I do not know, my guideline is: 
If it can crash due to user input, it should be a runtime_error
If it can only crash due to wrong programming it should be an assert

Reply:
`list1` and `list2` need to be in the same topology, so an assert makes sense here.